splinepy/io/gismo.py

```diff
@@ -190,9 +220,8 @@ def export(
     export_fields : bool
       Export fields to separate files ending with field<id>.xml, e.g.,
       filename.xml.field1.xml
-    collapse_fields : cool
-      Only valid if export_fields is True, writes all fields in the same file
-      by collapsing the control points, requires conformity (not checked)
+    as_base64 : bool (False)
+      If set, coordinates, weights and knot-vectors are exported in B64 ecoding
```

Comment:
```suggestion
      If set, coordinates, weights and knot-vectors are exported in B64 encoding.
      Available in gismo, once gismo/gismo#634 merges.
```
I think with this note, we can merge this. You could update readme too if you want